src/executors/default-with-elasticsearch.yml

```diff
@@ -1,3 +1,6 @@
+description: >
+  This executor is based on golang Docker image and has a
```

Comment:
**thought/super nit-pick (non-blocking):** Aren't those images running alongside each other ? In other words I think that docker will spin up two containers instead of one container which is build from Go & ES images.

Reply:
I agree that this description is not clear at all. I was trying to refer to the concept of [primary container](https://circleci.com/docs/glossary/#primary-container) which I got from the description of [the docker executor](https://circleci.com/docs/configuration-reference/#executor-job).
I hope the description is not too simplified and misleading.